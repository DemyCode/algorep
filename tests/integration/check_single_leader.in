0
3
no
timeout 1
get_state 1
get_state 2
get_state 3
stop 1
stop 2
stop 3