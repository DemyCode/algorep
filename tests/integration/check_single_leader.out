State 1: LEADER
State 2: FOLLOWER
State 3: FOLLOWER